src/libsystemd-network/sd-lldp.c

```diff
@@ -221,15 +215,14 @@ int lldp_handle_packet(tlv_packet *tlv, uint16_t length) {
                 case LLDP_TYPE_CHASSIS_ID:
 
                         if (len < 2) {
-                                log_lldp("Received malformed Chassis ID TLV len = %d. Dropping",
-                                         len);
+                                log_lldp("Received malformed Chassis ID TLV len = '%d'. Dropping.", len);
 
                                 malformed = true;
                                 goto out;
                         }
 
                         if (chassis_id) {
-                                log_lldp("Duplicate Chassis ID TLV found. Dropping ...");
+                                log_lldp("Duplicate Chassis ID TLV found. Dropping.");
```

Comment:
I don't get it. You have `tlv` and `port` as local variables, and both contain important information about the context of the packet. Why not log them if there's something wrong with the packet you're parsing? Again: what benefit for the administrator does the log message have as it currently stands?